src/Dialect/ONNX/ONNXOps.td

```diff
@@ -74,15 +74,16 @@ def ONNXEntryPointOp: ONNX_Op<"EntryPoint"> {
     The "onnx.EntryPoint" function indicates the main entry point of ONNX model.
   }];
 
-  let builders = [OpBuilderDAG<(ins "FuncOp":$function, "int":$numInputs, "int":$numOutputs)>];
+  let builders = [OpBuilderDAG<(ins "FuncOp":$function, "int":$numInputs, "int":$numOutputs, "std::string":$signature)>];
```

Comment:
Minor, if you could format to roughly be 80 chars

Reply:
done